opm/simulators/timestepping/SimulatorReport.cpp

```diff
@@ -73,103 +74,112 @@ namespace Opm
             if (failureReport) {
-                os << " (Failed: " << failureReport->update_time << "; "
-                   << 100*failureReport->update_time/t << "%)";
+	      os << fmt::format(" (Failed: {:2.1f}; {:2.1f}%)",
+				failureReport->update_time,
+				100*failureReport->update_time/t);
             }
             os << std::endl;
 
-            t = output_write_time + (failureReport ? failureReport->output_write_time : 0.0);
-            os << " Output write time (seconds): " << t;
+            os << fmt::format(" Output write time (seconds): {:6.1f}", 
+			      output_write_time + (failureReport ? failureReport->output_write_time : 0.0));
             os << std::endl;
 
         }
 
-        int n = total_well_iterations + (failureReport ? failureReport->total_well_iterations : 0);
-        os << "Overall Well Iterations:      " << n;
+	int n = total_well_iterations + (failureReport ? failureReport->total_well_iterations : 0);
+
+	os << fmt::format("Overall Well Iterations:      {:6}", n);
         if (failureReport) {
-            os << " (Failed: " << failureReport->total_well_iterations << "; "
-               << 100.0*failureReport->total_well_iterations/n << "%)";
+	  os << fmt::format(" (Failed: {:3}; {:2.1f}%)",
+			    failureReport->total_well_iterations,
+			    100.0*failureReport->total_well_iterations/n);
         }
         os << std::endl;
 
-        n = total_linearizations + (failureReport ? failureReport->total_linearizations : 0);
-        os << "Overall Linearizations:       " << n;
+	n = total_linearizations + (failureReport ? failureReport->total_linearizations : 0);
+        os << fmt::format("Overall Linearizations:       {:6}", n);
         if (failureReport) {
-            os << " (Failed: " << failureReport->total_linearizations << "; "
-               << 100.0*failureReport->total_linearizations/n << "%)";
+	  os << fmt::format(" (Failed: {:3}; {:2.1f}%",
+			    failureReport->total_linearizations,
+			    100.0*failureReport->total_linearizations/n);
         }
         os << std::endl;
 
         n = total_newton_iterations + (failureReport ? failureReport->total_newton_iterations : 0);
-        os << "Overall Newton Iterations:    " << n;
+        os << fmt::format("Overall Newton Iterations:    {:6}", n);
         if (failureReport) {
-            os << " (Failed: " << failureReport->total_newton_iterations << "; "
-               << 100.0*failureReport->total_newton_iterations/n << "%)";
+	  os << fmt::format(" (Failed: {:3}; {:2.1f}%",
+			    failureReport->total_newton_iterations,
+			    100.0*failureReport->total_newton_iterations/n);
         }
         os << std::endl;
 
-        n = total_linear_iterations + (failureReport ? failureReport->total_linear_iterations : 0);
-        os << "Overall Linear Iterations:    " << n;
+	n = total_linear_iterations + (failureReport ? failureReport->total_linear_iterations : 0);
```

Comment:
Nitpick: please replace tabs with spaces (it makes it unnecessary hard to read diffs).

Reply:
Sure, will do.